src/services/activityReports.js

```diff
@@ -1043,6 +1042,21 @@ export async function setStatus(report, status) {
  * @returns {*} Grants and Other entities
  */
 export async function possibleRecipients(regionId, activityReportId = null) {
+  // If we have an activityReportId get the startDate of the report.
+  // We have to consider that we have a report with no ActivityRecipients (ie no way to join).
+  const reportWithStartDate = activityReportId ? await ActivityReport.findOne({
+    attributes: ['startDate'],
+    where: {
+      id: activityReportId,
+    },
+  }) : null;
+
+  // If we have a valid startDate subtract 61 days from it.
+  let startDate = null;
+  if (reportWithStartDate && reportWithStartDate.startDate) {
+    startDate = moment(reportWithStartDate.startDate).subtract(61, 'days').format('YYYY-MM-DD');
+  }
```

Comment:
why not do this all in sql 
```suggestion
    const reportWithStartDate = activityReportId ? await ActivityReport.findOne({
      attributes: [
        'startDate',
        [sequelize.literal(`"startDate" - INTERVAL '61 days'`), 'adjustedStartDate']
      ],
      where: {
        id: activityReportId,
      },
    }) : null;

  // If we have a valid startDate subtract 61 days from it.
  const startDate = reportWithStartDate && reportWithStartDate.startDate
      ? reportWithStartDate.get('adjustedStartDate')
      :  null;
```

Reply:
I did initially we don't have a link to that report. Right now reports are joined through ActivityRecipients table we need to handle the case were a inactive grant is NOT linked to an AR.

Reply:
just remove this whole block